scripts/replace_validator.py

```diff
@@ -106,41 +194,18 @@ def main():
         if target_operator_account["pub_key"]:
             new_pubkey = {
                 "@type": "/cosmos.crypto.secp256k1.PubKey",
-                "key": args.dest_validator_operator_pubkey,
+                "key": dest_validator_operator_pubkey,
             }
             target_operator_account["pub_key"] = new_pubkey
 
-    # Brute force replacement of all remaining occurrences
-    genesis_dump = json.dumps(genesis)
-
-    # Convert validator valcons address
-    genesis_dump = re.sub(
-        target_consensus_address, dest_consensus_address, genesis_dump
-    )
-
-    # Convert operator valoper address
-    genesis_dump = re.sub(
-        target_operator_address,
-        dest_operator_valoper_address,
-        genesis_dump,
-    )
-
-    # Convert operator base account address
-    genesis_dump = re.sub(
-        target_operator_base_address,
-        dest_operator_base_address,
-        genesis_dump,
-    )
-
-    genesis = json.loads(genesis_dump)
```

Comment:
This was replaced due to potential risk replacing something what was not supposed to be replaced + performance impact.